specification/cognitiveservices/data-plane/ImmersiveReader/stable/v1.0/ImmersiveReader.json

```diff
@@ -0,0 +1,83 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Immersive Reader API v1.0",
```

Comment:
I know you don't need this to generate SDK, But I still recommend to keep consistent with other CS swaggers.
`"title": "Immersive Reader Client"`

Reply:
Title updated

Reply:
How do I get signoff from the API review board?

I had an API Design review yesterday, but the members in attendance didn't feel a review was warranted since this was an internal API.